files/perf-fio/run_fio_pod.sh.in

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
You pulled this file into your PR by accident.    This is part of the new fio automation